### ISSUE TITLE
Accessibility issues in the JS-generated ESDC site wrapper

### ISSUE THREAD

**Issue by Jefferydo** [bati-itao/bati-itao.github.io#1151]
I’ve noticed a bunch of accessibility issues in the JS-generated ESDC site wrapper. Most of these issues are in the header element, which follows the skip links.

1. Multiple nav elements without labels:
  - Main menu (mobile)
  - Main menu (desktop)
  - Breadcrumbs
  - Footer
2. The main menu ul has role=”menubar” but the pattern is incomplete/incorrect:
  - The child li elements are missing role=”none”.
  - The menuitem role is missing aria-expanded
    - Instead, the hidden submenu uses aria-expanded. It’s supposed to be set on the control, not the target.
3. The mobile menu has issues:
  - Multiple nav elements are used to build a single nav region with nav elements nested inside nav elements
  - Redundant ARIA roles (e.g., nav with role=”navigation”)
  - One too many "Menu" headings. The mobile nav element opens with two visually-hidden h2 "Menu" headings.
  - Mobile menu’s close button tab order is wrong; button needs to be moved to the top of the menu to match reading order
